service/docs/source/runtime.rst

```diff
@@ -128,16 +128,20 @@ Runtime launch options.
      getting launched, the optional ``GEOPM_PROGRAM_FILTER`` environment variable 
      can be set to explicitly list the program invocation name of the specific 
      non-MPI process that needs to be profiled.
+   * While optional in this example, in case of launching non-MPI applications across
```

Comment:
Same note as before. Maybe a link to the [GEOPM environment variables](https://geopm.github.io/geopm.7.html#geopm-environment-variables) documentation would be helpful somewhere (maybe after this bullet list. Same suggestion applies to the prior comment.